### DISCUSSION TITLE
CHANGEMENT DE LA LANGUE PAR DÉFAUT

### DISCUSSION THREAD

**Discussion by Haks-De** [termux/termux-app#3832] [Q&A] [answered]
_**Salut tous, dites moi comment chargée la langue par défaut du système (Anglais) en Français ?? Merci bien d'avance pour vos réponses la team 🌟.**_
![Screenshot_20240211-095056](https://github.com/termux/termux-app/assets/135075004/ff45d0d1-3c68-4a3c-b6bb-5fb7e7932997)

**Comment by twaik** [comment, marked as answer]
1. Official language of all termux repos is English.
2. Currently termux is not localized.

**Reply by Haks-De** [reply]
Donc je peux pas personnalisé la langue pour pouvoir lire et bien comprendre les codes sources de leur fichier !!! 🥺🥲?

**Reply by twaik** [reply]
Again. Official language here is English. We do not offer support in other languages.

**Reply by Haks-De** [reply]
Ok, d'accord. Je vais donc faire avec comme ça anh, 🥲
Merci bien. 

Aussi, dit moi, t'as une idée de comment je pourrais créer une base de données depuis termux et lié à ma page ?? (principalement j'ai le fichier ".php" quoi. 
Par exemple, j'ai ma page (formulaire) là (10.53.81.43:8080) et j'aimerais gérer les données qui y seront renseigné dans la DB à chaque visite quoi. 

Merci bien d'avance pour ta réponse 😉

**Reply by twaik** [reply]
Again. Official language here is English. We do not offer support in other language. French is not English.

**Reply by Haks-De** [reply]
Alright Alright. So I'm going to deal with it like that anh, 🥲 Thank you very much. Also, tell me, do you have an idea for a comment I could create a database from termux and linked to my page?? (mainly I have the ".php" file. For example, I have my page (form) there (10.53.81.43:8080) and I would like to manage the data that will be entered in the DB on each visit. Thank you in advance for your response

**Reply by twaik** [reply]
You can use standard set for this. Apache/Nginx + MariaDB/Sqlite or something.
Termux (in most cases) does not offer some exclusive tools for doing thing. It has the same tools as any other Linux distro, patched to work in our (pretty much restricted) environment.

**Reply by Haks-De** [reply]
OK, thanks for suggestions. I'll try whith that,
